Clinical-Genomics/trailblazer
# upgrades obsolete dependencies

This PR fixes the obsolete dependencies in TB

How to test:
1. install on stage on clinical-db
1. login to TB-stage

Expected outcome: 
Analyses in progress are listed
